docs/home/howto/OS_PATCHING.md

```diff
@@ -1,53 +1,63 @@
 # Patching OS with running Epiphany components
 
-This guide describes steps you have to perform to patch RHEL systems in a way to don't interrupt working Epiphany components.
+This guide describes steps you have to perform to patch RHEL and Ubuntu systems in a way to don't interrupt working Epiphany components.
 
 ### Disclaimer
 
-We provide a recommended way to patch your RHEL systems. Before proceeding with patching the production environment we highly recommend patching your test cluster first.  
+We provide a recommended way to patch your RHEL and Ubuntu systems. Before proceeding with patching the production environment we highly recommend patching your test cluster first.  
 This document will help you decide how you should patch your OS. This is not a step-by-step guide.
 
 ### Requirements
 
 - The fresh, actual backup containing your all important data
+- Enabled official repository
+- Disabled Epiphany-provided repository
```

Comment:
Is anything required from user to enable/disable repositories? I see that this part is described below, but it's nice to have at least a reference here.

Reply:
No, it's not. Actually this is the default state after successful epi cluster deployment. I put this on top as requirements because it's quite important to not break the environment e.g. by accidentally upgrading epi components.

Reply:
But may be user needs to know that `this is the default state after successful epi cluster deployment`? I read this and the first thought that comes - check if requirements are met. But there is nothing more.